tests/conftest.py

```diff
@@ -56,6 +56,7 @@ def pytest_addoption(parser: Parser):
         "--openstack-clouds-yaml",
         action="store",
         help="The OpenStack clouds yaml file for the charm to use.",
+        default="",
```

Comment:
The default without  this would be `None` right?
Would changing to empty string have advantages?

Reply:
I kept it so that it stays as type string and doesn't go to `None`. The default behavior without this would be an error saying the input is not given for this parameter.